### PR TITLE
 Fixes #4542 - fix broken link + style fixes

### LINKED ISSUE #4542: See Also links are dead.
Writing a Script Workflow can be found at https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/jj574157(v=ws.11)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37f22b6b-0a8b-6621-400a-c35c61e106fd
* Version Independent ID: 87cc1dfa-5683-06f4-bb20-5e126e8fe2a7
* Content: [about_Foreach Parallel](https://docs.microsoft.com/en-us/powershell/module/psworkflow/about/about_foreach-parallel?view=powershell-5.1#feedback)
* Content Source: [reference/5.1/PSWorkflow/About/about_Foreach-Parallel.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/PSWorkflow/About/about_Foreach-Parallel.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixes #4542 - fix broken link + style fixes
Fixes [AB#1569283](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1569283)

Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/PSWorkflow/New-PSWorkflowExecutionOption.md [hunk 1]
```diff
@@ -359,14 +358,16 @@ Accept pipeline input: False
...
+### -PersistencePath
 
-Encrypts the data in the persistence store.
-Consider using this feature when storing persistence data in a network share.
-The default value is False.
+Specifies the location on disk where workflow state and data are stored. Storing the workflow state
+and data allows workflows to be suspended and resumed, and to recover from interruptions and network
+failures.
+
+The default value is $env:LocalAppData\Microsoft\Windows\PowerShell\WF\PS
```
Comment (bobbytreed) [inline reference/3.0/PSWorkflow/New-PSWorkflowExecutionOption.md]: Should this be in backticks?

File: reference/3.0/PSWorkflow/New-PSWorkflowSession.md [hunk 2]
```diff
@@ -24,52 +25,63 @@ New-PSWorkflowSession [[-ComputerName] <String[]>] [-Credential <Object>] [-Name
...
+    New-PSWorkflowSession -Name WorkflowSession -Credential Domain01\Admin01 -ThrottleLimit 150
 ```
 
-This command creates workflow sessions on the ServerNode01 and Server12 computers.
-The command uses the **Credential** parameter to run with the permissions of the domain administrator.
-
 The command uses the **ThrottleLimit** parameter to increase the per-command throttle limit to 150.
-This value takes precedence over the default throttle limit of 100 that is set in the Microsoft.PowerShell.Workflow session configuration.
+This value takes precedence over the default throttle limit of 100 that is set in the
+Microsoft.PowerShell.Workflow session configuration.
```
Comment (bobbytreed) [inline reference/3.0/PSWorkflow/New-PSWorkflowSession.md]: Bolded type names

File: reference/3.0/PSWorkflow/New-PSWorkflowSession.md [hunk 3]
```diff
@@ -312,13 +347,15 @@ Accept wildcard characters: False
...
 
-Uses the Secure Sockets Layer (SSL) protocol to establish a connection to the remote computer.
-By default, SSL is not used.
+Indicates that this cmdlet uses the Secure Sockets Layer (SSL) protocol to establish a connection to
+the remote computer. By default, SSL is not used.
 
-WS-Management encrypts all Windows PowerShell content transmitted over the network.
-UseSSL is an additional protection that sends the data across an HTTPS connection instead of an HTTP connection.
+WS-Management encrypts all Windows PowerShell content transmitted over the network. The **UseSSL**
+cmdlet is an additional protection that sends the data across an HTTPS connection instead of an HTTP
```
Comment (bobbytreed) [inline reference/3.0/PSWorkflow/New-PSWorkflowSession.md]: UseSSL cmdlet?

File: reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md [hunk 4]
```diff
@@ -31,17 +35,28 @@ ForEach -Parallel ($<item> in $<collection>)
...
+Like the ForEach statement in Windows PowerShell, the variable that contains
+collection `$<collection>` must be defined before the `ForEach -Parallel`
+statement, but the variable that represents the current item `$<item>` is
+defined in the `ForEach -Parallel` statement.
 
-The ForEach -Parallel construct is different from the ForEach keyword and the Parallel keyword. The ForEach keyword processes the items in the collection in sequence. The Parallel keyword runs commands in a script block in parallel. You can enclose a Parallel script block in a ForEach -Parallel script block.
+The `ForEach -Parallel` construct is different from the ForEach keyword and the
+Parallel keyword. The ForEach keyword processes the items in the collection in
+sequence. The Parallel keyword runs commands in a script block in parallel. You
+can enclose a Parallel script block in a ForEach -Parallel script block.
```
Comment (bobbytreed) [inline reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md]: ForEach -Parallel should be in backticks, as should ForEach Keyword.  Also Parallel isn't a keyword is it?

File: reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md [hunk 5]
```diff
@@ -31,17 +35,28 @@ ForEach -Parallel ($<item> in $<collection>)
...
 
-The target computers in a workflow, such as those specified by the PSComputerName workflow common parameter, are always processed in parallel. You do not need to specify the ForEach -Parallel keyword for this purpose.
+The target computers in a workflow, such as those specified by the
+PSComputerName workflow common parameter, are always processed in parallel. You
```
Comment (bobbytreed) [inline reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md]: parameter bolded PSComputername

File: reference/4.0/PSWorkflow/New-PSWorkflowExecutionOption.md [hunk 6]
```diff
@@ -359,14 +358,16 @@ Accept pipeline input: False
...
+### -PersistencePath
 
-Encrypts the data in the persistence store.
-Consider using this feature when storing persistence data in a network share.
-The default value is False.
+Specifies the location on disk where workflow state and data are stored. Storing the workflow state
+and data allows workflows to be suspended and resumed, and to recover from interruptions and network
+failures.
+
+The default value is $env:LocalAppData\Microsoft\Windows\PowerShell\WF\PS
```
Comment (bobbytreed) [inline reference/4.0/PSWorkflow/New-PSWorkflowExecutionOption.md]: backtick env variable

File: reference/4.0/PSWorkflow/New-PSWorkflowSession.md [hunk 7]
```diff
@@ -23,49 +24,64 @@ New-PSWorkflowSession [[-ComputerName] <String[]>] [-Credential <Object>] [-Name
 ```
 
 ## DESCRIPTION
-The **New-PSWorkflowSession** cmdlet creates a user-managed session ("PSSession") that is especially designed for running Windows PowerShell workflows.
-It uses the Microsoft.PowerShell.Workflow session configuration, which includes scripts, type and formatting files, and options that are required for workflows.
 
-You can use "New-PSWorkflowSession" or its alias, nwsn.
+The `New-PSWorkflowSession` cmdlet creates a user-managed session (**PSSession**) that is
+especially designed for running Windows PowerShell workflows. It uses the
+Microsoft.PowerShell.Workflow session configuration, which includes scripts, type and formatting
```
Comment (bobbytreed) [inline reference/4.0/PSWorkflow/New-PSWorkflowSession.md]: bold typename

File: reference/4.0/PSWorkflow/New-PSWorkflowSession.md [hunk 8]
```diff
@@ -23,49 +24,64 @@ New-PSWorkflowSession [[-ComputerName] <String[]>] [-Credential <Object>] [-Name
...
+
+```powershell
+"ServerNode01", "Server12" |
+    New-PSWorkflowSession -Name WorkflowSession -Credential Domain01\Admin01 -ThrottleLimit 150
+```
 
 The command uses the **ThrottleLimit** parameter to increase the per-command throttle limit to 150.
-This value takes precedence over the default throttle limit of 100 that is set in the Microsoft.PowerShell.Workflow session configuration.
+This value takes precedence over the default throttle limit of 100 that is set in the
+Microsoft.PowerShell.Workflow session configuration.
```
Comment (bobbytreed) [inline reference/4.0/PSWorkflow/New-PSWorkflowSession.md]: bold typename

File: reference/3.0/PSWorkflow/New-PSWorkflowExecutionOption.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/PSWorkflow/New-PSWorkflowExecutionOption.md]: Fixed

File: reference/3.0/PSWorkflow/New-PSWorkflowSession.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/PSWorkflow/New-PSWorkflowSession.md]: Fixed

File: reference/3.0/PSWorkflow/New-PSWorkflowSession.md (hunk 3 above)
Comment (sdwheeler) [reply reference/3.0/PSWorkflow/New-PSWorkflowSession.md]: fixed

File: reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md (hunk 4 above)
Comment (sdwheeler) [reply reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md]: fixed

File: reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md (hunk 5 above)
Comment (sdwheeler) [reply reference/4.0/PSWorkflow/About/about_Foreach-Parallel.md]: Fixed

File: reference/4.0/PSWorkflow/New-PSWorkflowExecutionOption.md (hunk 6 above)
Comment (sdwheeler) [reply reference/4.0/PSWorkflow/New-PSWorkflowExecutionOption.md]: fixed

File: reference/4.0/PSWorkflow/New-PSWorkflowSession.md (hunk 7 above)
Comment (sdwheeler) [reply reference/4.0/PSWorkflow/New-PSWorkflowSession.md]: fixed

File: reference/4.0/PSWorkflow/New-PSWorkflowSession.md (hunk 8 above)
Comment (sdwheeler) [reply reference/4.0/PSWorkflow/New-PSWorkflowSession.md]: fixed

**Pull request merged**